File: desktop/src/ru/codemonkeystudio/olld40/desktop/DesktopLauncher.java
package ru.codemonkeystudio.olld40.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import ru.codemonkeystudio.olld40.CMSGame;

public class DesktopLauncher {

    public static void main(String[] args) {

        LwjglApplicationConfiguration configuration = new LwjglApplicationConfiguration();
        configuration.width = 1366;
        configuration.height = 768;

        new LwjglApplication(new CMSGame(), configuration);

    }

}

File: desktop/src/ru/codemonkeystudio/olld40/desktop/DesktopLauncher.kt
package ru.codemonkeystudio.olld40.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import ru.codemonkeystudio.olld40.CMSGame

fun main(arg: Array<String>) {
    val config = LwjglApplicationConfiguration()

    config.fullscreen = true
    config.width = 1366
    config.height = 768

    LwjglApplication(CMSGame(), config)
}